package.json

```diff
@@ -37,7 +37,6 @@
   },
   "dependencies": {
     "classnames": "~2.2.6",
-    "core-js": "~2.6.5",
```

Comment:
can you also update the lock file?

Reply:
@ad1992 usually `package-lock.json` should not be committed for libraries, so I suggest to delete it instead.

Reply:
@ad1992 updated it anyway in case that was a blocker

Reply:
we commit package.json so that all the contributors have the same set up in local development